can/interfaces/ixxat/canlib.py

```diff
@@ -308,11 +336,10 @@ def recv(self, timeout=None):
                 _canlib.canChannelPeekMessage(self._channel_handle, ctypes.byref(self._message))
             except VCITimeout:
                 return None
+            except VCIRxQueueEmptyError:
+                return None
             except VCIError as e:
-                if (e.HRESULT == constants.VCI_E_RXQUEUE_EMPTY):
-                    return None
-                else:
-                    raise e
+                raise e

```

Comment:
Unless you want to add some logging in the event of a `VCIError` you don't need to catch and re-raise this.